seshdash/views.py

```diff
@@ -81,6 +89,19 @@ def index(request,site_id=0):
     # Create an object of the get_high_chart_date
     context_dict['high_chart']= get_high_chart_data(request.user,site_id,sites)
     context_dict['site_id'] = site_id
+
+    #Generate measurements in the time_series_graph
+    client=Influx('test_db')
```

Comment:
This value is hardcoded here
This is a very bad practice. leaving it blank will automatically take the db from the config file!